tests/unit/AgmExtension.spec.ts

```diff
@@ -1,68 +1,70 @@
+import AgmExtensionHelp from '@/components/AgmExtension/AgmExtensionHelp.vue'
 import { ConfirmDialog, NotEligibleExtensionDialog, PaymentErrorDialog } from '@/components/dialogs'
 import { Certify } from '@/components/common'
 import { ExpandableHelp } from '@bcrs-shared-components/expandable-help'
-import AgmExtensionHelp from '@/components/AgmExtension/AgmExtensionHelp.vue'
-import AboutTheBusiness from '@/components/AgmExtension/AboutTheBusiness.vue'
-import AgmExtensionEvaluation from '@/components/AgmExtension/AgmExtensionEvaluation.vue'
 import ExtensionRequest from '@/components/AgmExtension/ExtensionRequest.vue'
-import { LegalServices } from '@/services'
+import SbcFeeSummary from 'sbc-common-components/src/components/SbcFeeSummary.vue'
 import flushPromises from 'flush-promises'
-import mockRouter from './mockRouter'
-import VueRouter from 'vue-router'
-import { CorpTypeCd, FilingCodes } from '@/enums'
 
-// suppress various warnings:
-// - "Unknown custom element <affix>" warnings
-// - "$listeners is readonly"
-// - "Avoid mutating a prop directly"
-// ref: https://github.com/vuejs/vue-test-utils/issues/532
+// suppress warning "Unknown custom element <affix>" warnings
 Vue.config.silent = true
 
 Vue.use(Vuetify)
 const vuetify = new Vuetify({})
 
-const localVue = createLocalVue()
-localVue.use(VueRouter)
 setActivePinia(createPinia())
 const businessStore = useBusinessStore()
 const rootStore = useRootStore()
 
 describe('AGM Extension view', () => {
   let wrapper: any
 
-  beforeAll(() => {
+  beforeEach(() => {
     // init store
     rootStore.currentDate = '2023-11-06'
     businessStore.setLegalType(CorpTypeCd.BENEFIT_COMPANY)
     businessStore.setLegalName('My Benefit Company')
     businessStore.setIdentifier('BC1234567')
-    businessStore.setFoundingDate('2000-01-01T08:00:00-00:00')
+    businessStore.setFoundingDate('2000-01-01T08:00:00.000+00:00')
+    businessStore.setGoodStanding(true)
     rootStore.filingData = []
-  })
-
-  beforeEach(() => {
-    const $route = { params: { filingId: '0' } }
 
     // create local Vue and mock router
     const localVue = createLocalVue()
     localVue.use(VueRouter)
-    const $router = mockRouter.mock()
+    const router = mockRouter.mock()
+    router.push({ name: 'agm-extension', params: { filingId: '0' } })
 
     wrapper = mount(AgmExtension, {
-      mocks: { $route, $router },
+      localVue,
+      router,
       stubs: {
-        ConfirmDialog: true,
-        PaymentErrorDialog: true,
-        NotEligibleExtensionDialog: true,
+        // NOTE - we want to stub out some components and not others:
         AboutTheBusiness: true,
+        AgmExtensionEvaluation: true,
+        // AgmExtensionHelp: true,
+        // Certify: true,
+        ConfirmDialog: true,
+        // ExpandableHelp: true,
         ExtensionRequest: true,
-        AgmExtensionEvaluation: true
+        NotEligibleExtensionDialog: true,
+        PaymentErrorDialog: true,
+        SbcFeeSummary: true
```

Comment:
The "true" sub-components are stubbed. They have no functionality and they look like this:
```
<aboutthebusiness-stub data-v-29161980="" data="[object Object]" class="mt-6"></aboutthebusiness-stub>
```

THIS test suite doesn't care what is inside this sub-component (which should have its own test suite).

Other sub-components are not stubbed, and since this is a full-mount (not shallow mount), they are rendered and functional, which I either needed or take advantage of in this test suite.